### PR TITLE
Update AssemblyScript to 0.19.10

### PR DESCRIPTION
**Pull request by otaviopace**
This PR:

- Updates AssemblyScript dependency on `package.json`
- Updates codegen for the newer version
- Adds migration changes for apiVersion on subgraphs

What's missing:
- throw error when subgraph's `apiVersion` isn't the one with the newest `AssemblyScript` version

Related PRs:

- https://github.com/graphprotocol/graph-ts/pull/185
- https://github.com/graphprotocol/graph-node/pull/2571

### CODE REVIEW

**Comment by otaviopace** [comment]
I'll fix the tests soon, it seems that's because it's pointing to the wrong `graph-ts`. Let me just open that PR first 🙂

**Review by schmidsi — APPROVED** [review]
I'm not very deep into this codebase, but it looks good. Can't wait for this update actually.

File: src/compiler/asc.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const asc = require('assemblyscript/cli/asc')
+
+const createExitHandler = inputFile => () => {
+  throw new Error(`The AssemblyScript compiler crashed when compiling this file: '${inputFile}'
+Unfortunately this error has not been mapped yet.
```
Comment (leoyvens) [inline src/compiler/asc.js:5]: This error message is really nice and gentle! I'd remove this part:
> Unfortunately this error has not been mapped yet

It's not clear what it means and doesn't add much to the already long message.

**Review by leoyvens — CHANGES_REQUESTED** [review]
Is there anywhere where we validate the api version? The migration is nice but it's not guaranteed to work, we should have an error if we try to build with an api version < 0.0.5.

File: src/compiler/asc.js (hunk 1 above)
Comment (otaviopace) [reply src/compiler/asc.js:5]: Ok, I'll do it, thanks!

**Comment by otaviopace** [comment]
@leoyvens that's a great question (blocking apiVersion), I'll take a look where I can put this to be guaranteed to fail

File: src/compiler/asc.js (hunk 1 above)
Comment (otaviopace) [reply src/compiler/asc.js]: Fixed!

**Review by leoyvens — APPROVED** [review]

**Pull request merged**